app/controllers/admin/npcs_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::NpcsController < ApplicationController
+  before_action :require_admin
+
+  def index
+    @npcs = NpcModel.all
+  end
+end
```

Comment:
I just wanted there to be an index, and not a show, since that's not really necessary for an Admin to see ... maybe it could be a `nice to have` but it's not really needed in my mind.  If it's needed, it could just be handled by the `npc_models_controller` and an `unless` on the show page.